<?xml version="1.0" ?>
<launch xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="https://gist.githubusercontent.com/nalt/dfa2abc9d2e3ae4feb82ca5608090387/raw/roslaunch.xsd">
    <!-- Arguments -->
    <arg name="log_level" default="INFO" />
    <arg name="conf_location" default="~/.config/niryo"/>

    <!-- Launching serial number node -->
    <include file="$(find niryo_robot_iot)/launch/serial_number.launch" >
        <arg name="log_level" value="$(arg log_level)" />
    </include>

    <!-- Launching mqtt client node -->
    <include file="$(find niryo_robot_iot)/launch/mqtt_client.launch">
        <arg name="log_level" value="$(arg log_level)" />
        <arg name="conf_location" value="$(arg conf_location)"/>
    </include>
</launch>
